EDDiscovery/EDDiscoveryForm.cs

```diff
@@ -679,6 +700,19 @@ public void LogText(string text, Color col)
             }
         }
 
+        public void LogLine(string text)
+        {
+            try
+            {
+                Invoke((MethodInvoker)delegate
+                {
+                    TravelHistoryControl.LogText(text + Environment.NewLine);
+                });
+            }
+            catch
+            {
+            }
```

Comment:
Suggestion: I think we would be better off not eating the exceptions on these log methods. The chance of an unexpected failure is low here I think. But if something bad does happen, it would probably better to learn of it sooner rather eventually realizing that the logging has stopped. I've left it intact for now though.